Resources/Prototypes/_Impstation/Entities/Clothing/Neck/cloaks.yml

```diff
@@ -80,3 +80,12 @@
     sprite: _Impstation/Clothing/Neck/Cloaks/hosgreatcoat.rsi
   - type: StealTarget
     stealGroup: HeadCloak
+
+- type: entity
+  parent: ClothingCloakCmo
+  id: ClothingNeckCloakCmosgreatcloak
```

Comment:
can you put this in pascal case to match other item ids? i.e. `ClothingNeckCloakCmoGreatcloak`